eZ/Publish/API/Repository/Tests/SearchService/RemoteIdIndexingTest.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\API\Repository\Tests\SearchService;
+
+use eZ\Publish\API\Repository\Tests\BaseTest;
+use eZ\Publish\API\Repository\Values\Content\Query;
+use eZ\Publish\API\Repository\Values\Content\Query\Criterion;
+
+final class RemoteIdIndexingTest extends BaseTest
+{
+    private const REMOTE_IDS = ['external:10', 'external_10', 'with space'];
+
+    /** @var \eZ\Publish\API\Repository\Tests\RepositoryTest */
+    private static $sharedRepository;
```

Comment:
This property seems to be redundant. See https://gist.github.com/adamwojs/0b4e3916d2e20d265c0833eeac7c7c1b

Reply:
Fixed via fb069b5